ci-operator/step-registry/ipi/conf/aws/sharednetwork/ipi-conf-aws-sharednetwork-ref.yaml

```diff
@@ -1,9 +1,6 @@
 ref:
   as: ipi-conf-aws-sharednetwork
-  from_image:
-    namespace: ocp
-    name: "4.12"
-    tag: upi-installer
+  from: upi-installer
```

Comment:
+ @patrickdillon @r4f4

Reply:
@jianlinliu Do you remember if there was any specific reason to use the 4.12 image?

Reply:
I don't think it would work. That means "take upi-installer image from the payload" but its not included in released payloads, so this step won't work when running against released images

Reply:
> this step won't work when running against released images
@vrutkovs could you please elaborate on that part, and if possible provide example to test it, and/or alternatives to achieve the goal suggested in this PR?

Reply:
> Do you remember if there was any specific reason to use the 4.12 image

If I remember right, some job did not define upi-installer in the job config level, so have to set a specific tag (not have to limit to 4.12, other versions also work) for the image.

Reply:
@patrickdillon we were uncertain how to test payloads in this/release repo. Rafa [tried here](https://github.com/openshift/release/pull/45481#issuecomment-1805726861)